src/Views/AccountView.vala

```diff
@@ -205,9 +205,9 @@ public class Installer.AccountView : AbstractInstallerView {
             return true;
         } else {
             if (username_is_taken) {
-                username_error_revealer.label = _("Username is already taken");
+                username_error_revealer.label = _("Chosen username already taken");
```

Comment:
Should this not be "The chosen usename has already been taken" if you want 'proper' grammar?

Reply:
It can be either, but it can't be the former.

Reply:
I agree, it is a balance between being succinct and being unambiguously translatable I guess.